report/test/renderer/details-renderer-test.js

```diff
@@ -608,6 +609,31 @@ describe('DetailsRenderer', () => {
           'did not style entity classified row as a grouped row');
       });
 
+      it('throws a warning for unsupported types being sorted', () => {
+        const backupConsole = global.console.warn;
+        global.console.warn = jestMock.fn();
+
+        renderer.render({
+          type: 'table',
+          sortedBy: ['totalBytes'],
+          headings: [
+            {key: 'url', valueType: 'url', label: 'URL'},
+            {key: 'totalBytes', valueType: 'text', label: 'Size (KiB)'},
+            {key: 'wastedBytes', valueType: 'bytes', label: 'Potential Savings (KiB)'},
+          ],
+          items: [
+            {url: 'https://example.com/1',
+              totalBytes: {type: 'link', text: 'linkText', url: 'linkUrl'},
+              wastedBytes: 500, entity: 'example.com'},
+            {url: 'Unattributable', totalBytes: true, wastedBytes: 500},
+            {url: 'https://cdn.com/2', totalBytes: 400, wastedBytes: 800, entity: 'cdn.com'},
+          ],
+        });
+
+        expect(global.console.warn).toHaveBeenCalled();
+        global.console.warn = backupConsole;
```

Comment:
I think you'll want to do this in an `after(() => ...)` clause in case `renderer.render` throws an error